tensorflow/c/BUILD

```diff
@@ -499,6 +499,7 @@ tf_cuda_library(
         ":tf_status",
         ":tf_status_helper",
         ":tf_tensor_internal",
+        "//tensorflow/c/experimental/stream_executor:stream_executor",
```

Comment:
Move this dep to be under //conditions:default below, so we don't include it for android.

Reply:
done